core/templates/dev/head/domain/collection/EditableCollectionBackendApiServiceSpec.js

```diff
@@ -107,9 +107,10 @@ describe('Editable collection backend API service', function() {
 
     collection.title = 'New Title';
     collection.version = '2';
+    collectionWrapper = {"collection" : collection}
```

Comment:
Hi @joshuacano -- if you can do a quick fix, we use single quotes in JS and we also don't put quotes around object keys in general. I.e. like this:

```
    collectionWrapper = {
      collection : collection
    };
```

Otherwise, I suspect it will fail the linter. Maybe run scripts/pre_commit_linter.py to double check.

Thanks!